src/test/java/com/julienviet/pgclient/PgConnectionTestBase.java

```diff
@@ -1060,6 +920,44 @@ public void testSetAndGetSerializableTx(TestContext ctx) {
     });
   }
 
+  @Test
+  public void testPgUpdate(TestContext ctx) {
```

Comment:
just testUpdate no need to say PgUpdate here

Reply:
The testUpdate will conflict with other test case so it's better to rename to testPreparedUpdate